modules/common/src/test/scala/com.snowplowanalytics.snowplow.rdbloader.common/ShredderConfigSpec.scala

```diff
@@ -67,6 +68,14 @@ class ShredderConfigSpec extends Specification {
       ) must beTrue
     }
 
+    "give error if option config list contains items with same type" in {
+      val result = getConfig("/test.config2.hocon", ShredderConfig.Batch.fromString)
+      result.fold(
```

Comment:
It might be able to express this like:

```
result must beLeft.like { err =>
  err must contain("Option config shouldn't contain items have same type")
}
```

or even

```
result must beLeft(contain("Option config shouldn't contain items have same type"))
```

[link](https://etorreborre.github.io/specs2/guide/SPECS2-4.10.0/org.specs2.guide.Matchers.html)